### ISSUE TITLE
Container: copy/pasting some content directly in source mode in main content breaks the platform 

### ISSUE THREAD

**Issue by nino-filigran** [LuatixHQ/opencti#8689]
## Description

In a container, when turning the main content into source mode & copy pasting some text, the platform breaks 

## Environment

demo octi 

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. go in a report
2. switch main content into source
3. copy/paste some text
4. platform hangs & breaks (page become unresponsive, impossible to do anything) 

Happens also when copy/pasting some text as source in Data/customization of the content 

## Expected Output

Like the UI mode, I expect  that the platform is able to copy/paste in source mode without breaking 

## Actual Output

Platform hangs & breaks (need to close the tab) 

## Additional information

text taken from: https://filigran.io/introducing-creation-relationships-entities-and-observables/#h-create-multiple-ips-linked-to-an-intrusion-set

Basically, this behavior happens each time you're in source mode, anywhere on the platform 

## Screenshots (optional)

Uploading platform break.mp4…

**Comment by nino-filigran** [comment]
Solved with new CKeditor version

**Comment by Archidoit** [comment]
Solved with CK Editor upgrade
